### PR TITLE
Poc release

### PR DESCRIPTION
**Pull request by TNO-SlaFleur**

### CODE REVIEW

**Comment by TNO-SlaFleur** [comment]
@jimrojerTNO Could you review this once you have accepted the invite to the Nieuwe-Warmte-Nu organization? I cannot add you as a reviewer as you are not a member yet. Invite should be in your inbox!

File: setup.py [hunk 1]
```diff
@@ -52,7 +61,7 @@
     ],
```
[the remark this answers is not in the document]
Comment (TNO-SlaFleur) [reply setup.py:61]: Latest version is 2.5.2 on pypi. If we up to 3.5 for rtc-tools then we will cause a dependency resolution error. Did you perhaps mean a different dependency?

[the remark this answers is not in the document]
Comment (jimrojerTNO) [reply setup.py:61]: I misread something on pypi. We need Casadi version==3.6.3 to support the new solver "highs". Casadi is normally installed via rtc-tools, but rtc-tools specifies version 3.5 for casadi which doesn't support the new solver.

rtc-tools version 2.5.2 works fine with the newer version of casadi, can we specifically define to use casadi 3.6.3 in our setup.py?

[the remark this answers is not in the document]
Comment (TNO-SlaFleur) [reply setup.py:61]: We cannot without changing the rtc-tools version. In 2.5.2 they specifically stated that is only works with Casadi `3.5.*` so if we add a dependency rule that we need Casadi `3.6.3` the dependency manager will not find a suitable Casadi version.

Checking `rtc-tools` I find that they have already released `2.6.*` prerelease versions. For instance, `2.6.0a3`. So if we change our `rtc-tools` version to `2.6.0a3` I expect we will pull in the necessary Casadi version.

Is it okay if we move to the newer `rtc-tools` version?

[the remark this answers is not in the document]
Comment (jimrojerTNO) [reply setup.py:61]: yes that should be fine!

[the remark this answers is not in the document]
Comment (TNO-SlaFleur) [reply setup.py:61]: Done: https://github.com/Nieuwe-Warmte-Nu/rtc-tools-heat-network/pull/2/commits/0c72d65e5d8e0d0a98b0dca9bb161ee59bc25ead

[the remark this answers is not in the document]
Comment (TNO-SlaFleur) [reply setup.py:63]: @jimrojerTNO  Update `rtc-tools` to `2.6.0a3` causes the unit tests to crash. I propose we do this in another MR where the appropriate unit tests fixes are also added. For now I have reverted the update and fixed all other issues that came loose when we updated to python3.8 minimum.

[the remark this answers is not in the document]
Comment (TNO-SlaFleur) [reply setup.py:63]: Issue is #7

**Review by jimrojerTNO — APPROVED** [review]

**Pull request merged**